# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
#
# Filename: package/.../linux26mm/hotfix_amso1100.patch
# Copyright (C) 2006 The OpenSDE Project
#
# More information can be found in the files COPYING and README.
#
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

  CC [M]  drivers/infiniband/hw/amso1100/c2_ae.o
  drivers/infiniband/hw/amso1100/c2_ae.c: In function 'c2_ae_event':
  drivers/infiniband/hw/amso1100/c2_ae.c:199: warning: implicit declaration of function 'to_event_str'
  drivers/infiniband/hw/amso1100/c2_ae.c:203: warning: implicit declaration of function 'to_qp_state_str'
  drivers/infiniband/hw/amso1100/c2_ae.c:203: warning: format '%s' expects type 'char *', but argument 3 has type 'int'
  drivers/infiniband/hw/amso1100/c2_ae.c:203: warning: format '%s' expects type 'char *', but argument 7 has type 'int'
  
WARNING: "to_qp_state_str" [drivers/infiniband/hw/amso1100/iw_c2.ko] undefined!
WARNING: "to_event_str" [drivers/infiniband/hw/amso1100/iw_c2.ko] undefined!

--- ./drivers/infiniband/hw/amso1100/c2_ae.c.orig	2006-10-09 06:47:05.000000000 -0400
+++ ./drivers/infiniband/hw/amso1100/c2_ae.c	2006-10-09 07:16:03.000000000 -0400
@@ -66,7 +66,6 @@
 	}
 }
 
-#ifdef DEBUG
 static const char* to_event_str(int event)
 {
 	static const char* event_str[] = {
@@ -144,7 +143,6 @@
 		return "<invalid QP state>";
 	};
 }
-#endif
 
 void c2_ae_event(struct c2_dev *c2dev, u32 mq_index)
 {
